core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -7448,27 +7448,6 @@ LogicalProject(SAL=[$5])
               LogicalProject(SAL=[$5], $f9=[=($5, 4)])
                 LogicalFilter(condition=[AND(=($7, 20), >($5, 1000))])
                   LogicalTableScan(table=[[CATALOG, SALES, EMPNULLABLES]])
-]]>
```

Comment:
where is the planAfter?

Reply:
After modification, this case becomes checkUnchanged.